Swift output fix

Zipfer opened the pull request:
BUG: https://bugs.launchpad.net/fuel/+bug/1248651
tested only with rspec

mirantis-jenkins commented:
Merged build triggered.

mirantis-jenkins commented:
Merged build triggered.

mirantis-jenkins commented:
Merged build started.

mirantis-jenkins commented:
Merged build finished.

mirantis-jenkins commented:
:+1: Test PASSed.
Refer to this link for build results: http://fuel-jenkins.mirantis.com/job/fuel_syntax_check/550/

mirantis-jenkins commented:
Merged build started.

mirantis-jenkins commented:
Merged build finished.

mirantis-jenkins commented:
:+1: Test PASSed.
Refer to this link for build results: http://fuel-jenkins.mirantis.com/job/fuel_pullrequest_systest/111/

dmitryilyin commented:
Parsing commands output with regeps is SOOO fun, especially when output format changes all the time and there is no rest api.

Zipfer committed:
Fix swift output parse regexp

 - Bug #1248651
 - fix parse output with new swift havana release
 - fix optional whitespace at the end of the line

Zipfer committed:
specs for new swift version

mirantis-jenkins commented:
Merged build triggered.

mirantis-jenkins commented:
Merged build started.

mirantis-jenkins commented:
Merged build triggered.

mirantis-jenkins commented:
Merged build finished.

mirantis-jenkins commented:
:+1: Test PASSed.
Refer to this link for build results: http://fuel-jenkins.mirantis.com/job/fuel_syntax_check/563/

mirantis-jenkins commented:
Merged build started.

mirantis-jenkins commented:
Merged build finished.

mirantis-jenkins commented:
:+1: Test PASSed.
Refer to this link for build results: http://fuel-jenkins.mirantis.com/job/fuel_pullrequest_systest/125/

mirantis-jenkins commented:
Merged build triggered.

mirantis-jenkins commented:
Merged build triggered.

mirantis-jenkins commented:
Merged build started.

mirantis-jenkins commented:
Merged build finished.

mirantis-jenkins commented:
:+1: Test PASSed.
Refer to this link for build results: http://fuel-jenkins.mirantis.com/job/fuel_syntax_check/570/

mirantis-jenkins commented:
Merged build started.

mirantis-jenkins commented:
Merged build finished.

mirantis-jenkins commented:
:+1: Test PASSed.
Refer to this link for build results: http://fuel-jenkins.mirantis.com/job/fuel_pullrequest_systest/132/

mirantis-jenkins commented:
Merged build triggered.

mirantis-jenkins commented:
Merged build started.

mirantis-jenkins commented:
Merged build triggered.

mirantis-jenkins commented:
Merged build started.

mirantis-jenkins commented:
Merged build finished.

mirantis-jenkins commented:
:+1: Test PASSed.
Refer to this link for build results: http://fuel-jenkins.mirantis.com/job/fuel_syntax_check/574/

mirantis-jenkins commented:
Merged build finished.

mirantis-jenkins commented:
:-1: Test FAILed.
Refer to this link for build results: http://fuel-jenkins.mirantis.com/job/fuel_pullrequest_systest/136/

aglarendil commented:
+2
